docs/ops.html

```diff
@@ -4182,6 +4182,14 @@ <h3>Reverting to ZooKeeper mode During the Migration</h3>
     If you did not fully complete any step, back out whatever you have done and then follow revert
     directions for the last fully completed step.
   </p>
+  <p>
+    In any case, once reverting is completed, you must clean-up any migration leftover before you attempt a new one
+  </p>
+  <ul>
+    <li>Remove <code>__cluster_metadata</code> logs directory from KRaft controllers</li>
+    <li>Remove <code>__cluster_metadata</code> logs directory from Zookeeper brokers</li>
```

Comment:
They'll be removed from the brokers automatically. The controllers are the ones that need to be manually removed. (We should probalby add a note about this.)